custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/MavenHelper.java

```diff
@@ -134,7 +134,8 @@ public void downloadJAR(File buildDir, DependencyInfo dep, String version, File
 
     public List<DependencyInfo> listDependenciesFromPom(File buildDir, File pom, File destination) throws IOException, InterruptedException {
         List<DependencyInfo> dependencies = new LinkedList<>();
-        int listed = run(buildDir, true, "dependency:list", "-B", "-DoutputFile=" + destination.getAbsolutePath(),  "-DincludeScope=runtime", "-DexcludeClassifiers=tests", "-f", pom.getAbsolutePath(), "-q");
+        LOGGER.log(Level.INFO, "Listing dependencies from POM file: {0}", pom);
+        int listed = run(buildDir, true, "dependency:list", "-B", "-DoutputFile=" + destination.getAbsolutePath(),  "-DincludeScope=runtime", "-DexcludeClassifiers=tests", "-f", pom.getAbsolutePath());
```

Comment:
What was the reason for the -q here?

Reply:
Log suppression. This step takes several minutes on integration tests, and it was just a radio silence